docs/examples/code/working_with_signed_urls.md

```diff
@@ -0,0 +1,135 @@
+
+**ref**: https://github.com/developmentseed/titiler/discussions/331
+
+**Goal**: allow users to pass signed url
+
+**requirements**: titiler.core
+
+## What / Why / How
+
+Passing a signed URL for a dataset is not supported by default in TiTiler because the parameters from the signed url are conflicting with the query parameters from the application itself. In order to allow signed url in the application there are two solutions:
```

Comment:
```suggestion
Passing a signed URL for a dataset is not supported by default in TiTiler because the parameters from the signed url conflict with the query parameters from the application itself. In order to allow signed url in the application there are two solutions:
```